### ISSUE TITLE
[BUG] RFT mode does not rename instance method properly when class is imported

### ISSUE THREAD

**Issue by hongphu8790** [dashingsoft/pyarmor#1369]
Assume we have two class Bar & Foo like this:

**foo.py**
```
class Foo:
    
    def __init__(self):
        self.name = "Foo"


    def say_name(self):
        return self.name
```

**bar.py**
```
from foo import Foo

class Bar:
    
    def __init__(self):
        self.foo = Foo()

    def say(self):
       return self.foo.say_name()
```

**Execute command**:
```
pyarmor cfg enable_trace=1 trace_rft=1
pyarmor cfg rft_auto_exclude=0
```

```
pyarmor -d gen --output ./dist --enable-rft *.py
INFO     Python 3.9.17
INFO     Pyarmor 8.2.9 (pro), 005235, <<removed>>
INFO     Platform linux.x86_64
DEBUG    native platform linux.x86_64
```

**Output**:

cat .pyarmor/rft/foo.py
```
class pyarmor__4:

    def __init__(self):
        self.pyarmor__6 = 'Foo'

    def pyarmor__5(self):
        return self.pyarmor__6
```

cat .pyarmor/rft/bar.py 
```
(pyarmor__4,) = __assert_armored__(b'\x84e\xbe\xf8U_9Q\xe7\xcb\xd8\xa4R\xb0\xf5-\xd7\x07{\x11')

class pyarmor__1:

    def __init__(self):
        self.pyarmor__3 = pyarmor__4()

    def pyarmor__2(self):
        return self.pyarmor__3.say_name()
```

Expected:

`self.pyarmor__3.say_name()` will be `self.pyarmor__3.pyarmor__5()`

All instance function does not renamed, that's problem of RFT or missing config. Thanks!

**Comment by jondy** [comment]
Reproduced, it will be fixed in next release.

As a quick workaround, use this undocumented configuration:
```
pyarmor cfg  -p bar rft_rulers ^ "self.foo.* self.?.?"
```

It means in module `bar` each pattern `self.foo.*` will be renamed to `self.X.X`

**Comment by hongphu8790** [comment]
> Reproduced, it will be fixed in next release.
> 
> As a quick workaround, use this undocumented configuration:
> 
> ```
> pyarmor cfg  -p bar rft_rulers ^ "self.foo.* self.?.?"
> ```
> 
> It means in module `bar` each pattern `self.foo.*` will be renamed to `self.X.X`

Yes, but a lot of classes methods, we can't do by rule by rules, waiting for next version. This is most important for RFT

**Comment by jondy** [comment]
Fixed in 8.3.dev1

**Comment by hongphu8790** [comment]
I should reopen this case for advanced:

foo.py
```
class Foo:

    def __init__(self, name):
        self.name = name

    def say_name(self):
        return self.name
```

bar.py
```
from foo import Foo

class Bar:

    def __init__(self, foo: Foo):
        self.foo = foo

    def say(self):
        return self.foo.say_name()
```

Same obfuscation command, output:

.pyarmor/rft/foo.py
```
class pyarmor__4:

    def __init__(self, name):
        self.pyarmor__6 = name

    def pyarmor__5(self):
        return self.pyarmor__6
```

.pyarmor/rft/bar.py
```
pyarmor__4, = __assert_armored__(b'\x84e\xbe\xf8U_9Q\xe7\xcb\xf8\xadd\xa6\xfb\xb6\xbah\tN\xca\x01h\xf9\xfc\xf7\xbb')

class pyarmor__1:

    def __init__(self, foo: pyarmor__4):
        self.pyarmor__3 = foo

    def pyarmor__2(self):
        return self.pyarmor__3.say_name()
```

Expected: `self.pyarmor__3.say_name()` should be `self.pyarmor__3.pyarmor__5()`

**Comment by jondy** [comment]
Arguments aren't renamed by rft mode, local variables are renamed.

This bug is missing types in arguments, it will be fixed in next release.

**Comment by hongphu8790** [comment]
@jondy Have another problem so funny,

bar.py
```
class BBar:

    def say(self):
        pass
```

foo.py
```
from bar import BBar

a = BBar()
a.say()
```

```
 pyarmor -d gen --output ./dist --enable-rft  *.py

INFO     Python 3.11.4
INFO     Pyarmor 8.3.0 (pro), 005235, <<removed>>
INFO     Platform linux.x86_64
DEBUG    native platform linux.x86_64
```

.pyarmor/rft/bar.py

```
class pyarmor__1:

    def pyarmor__2(self):
        pass
 ````
 
 
.pyarmor/rft/foo.py
```
pyarmor__1, = __assert_armored__(b'\x84e\xbe\xf8U[7L\xe7\xcb\xd8\xadd\xa6\xfb\xb6\xbah\tN\xca\x04h\xf9\xfc\xf7\xbb')
pyarmor__3 = pyarmor__1()
pyarmor__3.say()
```

Expected: `pyarmor__3.say()` should be `pyarmor__3.pyarmor__2()`

But, when you rename class BBar to Bar should be fine :)

Working: Bar, Barz, Barzz, Barzzz
Non-working: BBar, BaBar, BarzBar,...

**Comment by jondy** [comment]
Thanks, it will be fixed in next release.

**Comment by jondy** [comment]
Fixed in 8.3.1
